QtCollider/primitives/prim_misc.cpp

```diff
@@ -167,7 +167,7 @@ QC_LANG_PRIMITIVE( Qt_SetGlobalPalette, 1, PyrSlot *r, PyrSlot *a, VMGlobals *g
 {
   if( !QcApplication::compareThread() ) return QtCollider::wrongThreadError();
 
-  QPalette p = QtCollider::get( a );
+  QPalette p = (QPalette&&) QtCollider::get(a);
```

Comment:
with a raw c-style cast anything can be made to compile, in this case it looks super-dangerous.

maybe something like this works?
`static_cast<QPalette>( QtCollider::get( a ) );`